docs/install/authentication-machanisms-gcp.md

```diff
@@ -49,12 +49,24 @@ gcloud projects add-iam-policy-binding $PROJECT_ID \
 
 1. Enable Workload Identity.
 
-    ```shell
-    gcloud services enable iamcredentials.googleapis.com
-    gcloud beta container clusters update $CLUSTER_NAME \
-    --identity-namespace=$PROJECT_ID.svc.id.goog
-    ```
-
+    1. Ensure that you have enabled the Cloud IAM Service Account Credentials API.
+        ```shell
+        gcloud services enable iamcredentials.googleapis.com
+        ```
+    1. If you didn't enable Workload Identity when you
+    created your cluster, run the following commands: 
+    
+        ```shell 
+         gcloud container clusters update $CLUSTER_NAME \
+            --workload-pool=$PROJECT_ID.svc.id.goog
+         gcloud container node-pools update default-pool \
```

Comment:
Can we get all node pools and update them? Or at least make `default-pool` a variable.